target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -99,6 +99,16 @@ define Device/tplink_tl-wr1043nd-v1
 endef
 TARGET_DEVICES += tplink_tl-wr1043nd-v1
 
+define Device/tplink_tl-wr710n-v1
+  $(Device/tplink-8mlzma)
+  ATH_SOC := ar9331
+  DEVICE_TITLE := TP-Link TL-WR710N v1
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb-chipidea2 kmod-usb-ledtrig-usbport
```

Comment:
> swconfig is needed for VLANs on LAN interface.

Ok, this smells a bit broken. Nowadays, it should be possible to use the kernel's generic vlan
tagging/untagging support to do that. 

Try:
```
     # ip link add link eth0 name eth0.1 type vlan id 1 
     # ip link show
```

This will create a new  virtual lan ethernet device for vlanid 1 on eth0.